pkg/nuctl/command/export.go

```diff
@@ -53,9 +55,15 @@ func newExportFunctionCommandeer(exportCommandeer *exportCommandeer) *exportFunc
 	}
 
 	cmd := &cobra.Command{
-		Use:     "functions [name[:version]]",
-		Aliases: []string{"fu", "fn", "function"},
-		Short:   "(or function) Export function to yaml format",
+		Use:     "functions [<function>]",
+		Aliases: []string{"function", "fn", "fu"},
+        Short:   "(or function) Export functions",
+		Long:    "(or function) Export the configuration of a specific function or of all deployed Nuclio functions (default)
```

Comment:
```suggestion
		Use:     "functions [<function>]",
		Aliases: []string{"function", "fn", "fu"},
		Short:   "(or function) Export functions",
		Long:    "(or function) Export the configuration of a specific function or of all deployed Nuclio functions (default)
```
Inconsistent indentation here as well, please align the `Short` member to the rest of the members - lint issue.